clients/fides-js/__tests__/lib/i18n/i18n-utils.test.ts

```diff
@@ -490,14 +603,24 @@ describe("i18n module", () => {
         expect(testI18n.t({ id: "test.greeting" })).toEqual("Zalloz, Jest!");
         expect(testI18n.t({ id: "test.another" })).toEqual("Zam!");
       });
+
+      it("treats null/empty strings as missing keys", () => {
+        testI18n.load("zz", { "test.empty": "", "test.null": null } as any);
+        testI18n.activate("zz");
+        expect(testI18n.t({ id: "test.empty" })).toEqual("test.empty");
+        expect(testI18n.t({ id: "test.null" })).toEqual("test.null");
+        expect(testI18n.t({ id: "test.missing" })).toEqual("test.missing");
+      });
     });
 
-    it("treats null/empty strings as missing keys", () => {
-      testI18n.load("zz", { "test.empty": "", "test.null": null } as any);
-      testI18n.activate("zz");
-      expect(testI18n.t({ id: "test.empty" })).toEqual("test.empty");
-      expect(testI18n.t({ id: "test.null" })).toEqual("test.null");
-      expect(testI18n.t({ id: "test.missing" })).toEqual("test.missing");
+    describe("locale getter", () => {
+      it("returns the currently active locale", () => {
+        expect(testI18n.locale).toEqual("en");
+        testI18n.load("zz", { "test.greeting": "Zalloz, Jest!" });
+        expect(testI18n.locale).toEqual("en");
+        testI18n.activate("zz");
+        expect(testI18n.locale).toEqual("zz");
+      });
     });
   });
 });
```

Comment:
Nice! I will test against all these cases. Let me know if there are others I should test for UAT @NevilleS

Reply:
Nothing super specific... I think we really just should do a ton of exploratory testing around things and look to see if we can ever get a untranslated string to appear 👍